src/components/vote/QueueModal.tsx

```diff
@@ -0,0 +1,153 @@
+import { Trans } from '@lingui/macro'
```

Comment:
See all comments for `ExecuteModal.tsx`.
Also, please open up a jira issue to refactor the `src/components/vote/*Modal.tsx` files to use a shared `Modal.tsx` base component.

Reply:
https://uniswaplabs.atlassian.net/browse/WEB-171